.markdownlint-cli2.jsonc

```
@@ -0,0 +1,9 @@
+{
+  // Disable some built-in rules
+  "config": {
+    "line-length": false
```

The main reason I looked into pulling markdownlint early into SGX is that the README.mds in the mobilecoin repo were inconsistent in regards to line lengths.  Some paragraphs were 200+characters long while some were wrapped at around 80.
  
I can understand if we feel that 80 is too narrow, but perhaps being more explicit on what we want is better than blanket disabling
```suggestion
    "line-length": {"line_length": 100}
```

FYI: @remoun as they had interest in markdownlint so might be interested in the outcome of this discussion.

---

Right, my preference is "no limit"/"soft wrap" :-)

---

Doing some googling it seems that hard vs soft wrapping in markdown (and latex) files seems to be a religious debate with no clear winner.  My preference for hard wrapping is probably influenced by vim usage and easy reformatting via `gqq`.  Unfortunately not everyone understands that Vim is the best editor experience ever :P.  

Lacking a good argument that isn't preference, I have no more objections to disabling the line length check.